application.py

```diff
@@ -40,7 +40,7 @@ def index():
 @app.route('/resubmit', methods=['POST'])
 def resubmit():
     if request.method == 'POST':
-        r = requests.post(app.config['CODE_RESUBMISSION'],
+        r = requests.post('{}?language={}'.format(app.config['CODE_RESUBMISSION'], request.args.get('language')),
```

Comment:
Quick suggestion here, we should not use query parameters for changing the languages as it can be easily fiddled with. How about we pass it in the json? It is much safer that way.

Reply:
If we go with what I said, we will have to change the frontend flask server accordingly.

Reply:
I tried that, the issue is that you cant post files and json at the same time.

Reply:
Did you try adding key-value pairs to the FormData?

Reply:
Nope but I cant be bothered to do that now :) Lets just merge this and deal with it later. Spent the whole day doing this.